websites/D/Deno/presence.ts

```diff
@@ -2,71 +2,214 @@ const presence = new Presence({
 		clientId: "843058683100266526",
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
-let moduleName: HTMLElement;
 
-// checkmate javascript
-function pathIncludes(string: string): boolean {
-	return document.location.pathname.toLowerCase().includes(string);
+enum Assets {
+	Logo = "https://i.imgur.com/LF4eDYG.png",
+	NewsLogo = "https://i.imgur.com/aBbfq11.png",
+	MerchLogo = "https://i.imgur.com/GOPd2Ep.png",
+	SearchImage = "https://i.imgur.com/oGQtnIY.png",
+	ReadingImage = "https://i.imgur.com/nese1O7.png",
 }
 
 presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
-		largeImageKey: "logo",
-		startTimestamp: browsingTimestamp,
-	};
+			largeImageKey: Assets.Logo,
+			startTimestamp: browsingTimestamp,
+		},
+		{ href, hash, hostname, pathname } = document.location,
+		[privacy, buttons] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+		]),
+		search = document.querySelector<HTMLInputElement>('[id="search-input"]'),
+		pathnameSplit = pathname.split("/");
+	if (privacy) {
+		presenceData.largeImageKey = Assets.Logo;
+		presenceData.details = "Browsing";
+		if (presenceData.details) presence.setActivity(presenceData);
```

Comment:
```suggestion
		presence.setActivity(presenceData);
```
Useless if since you're setting a string just before, so the if is always true.